python_modules/dagster/dagster/_core/execution/job_backfill.py

```diff
@@ -227,7 +255,7 @@ def _get_partitions_chunk(
             if partition_name not in completed_partitions
         ]
 
-    return to_submit, next_checkpoint, has_more
+    return to_submit, next_checkpoint or "", has_more
```

Comment:
nit: next_checkpoint should always be something at this point, right? can we remove the `or ""`?

Reply:
I think it should, but pyright didn't believe it